Formula/q.rb

```diff
@@ -22,8 +22,12 @@ class Q < Formula
   def install
     # broken symlink, fixed in next version
     rm_f "bin/qtextasdata.py"
-
     virtualenv_install_with_resources
+
+    ENV["GEM_HOME"] = ".gem_home"
+    system "gem", "install", "ronn"
```

Comment:
This needs to be declared as a `resource` so that it's versioned and checksummed. You can check the other formulae that install Ruby gems for examples.

It's probably also simpler to use `pandoc` to convert the usage doc into a manpage.

Reply:
Actually, I'm not the one that came up with this _trick_, previous examples:
- `git-lfs` https://github.com/Homebrew/homebrew-core/blob/e863b0f42c1b98f5dd3814a42852948f1b35cb1a/Formula/git-lfs.rb#L24-L34
- `libidn2` https://github.com/Homebrew/homebrew-core/blob/e863b0f42c1b98f5dd3814a42852948f1b35cb1a/Formula/libidn2.rb#L36-L39

In both cases `ronn` isn't marked as a resource.

> It's probably also simpler to use `pandoc` to convert the usage doc into a manpage.

I agree but that is what upstream uses: https://github.com/harelba/q/blob/5f0c650a540059f17846c543d68f384178b9d02f/dist/q-text-as-data.spec.template#L26-L28

Reply:
@carlocab How about solving it by introducing new `ronn` formula and declaring it as a build dependency for listed formulas? 💡

Reply:
`git-lfs` and `libidn2` shouldn't be doing that. If I were reviewing those changes, I'd say the same thing.

We should fix `git-lfs` and `libidn2` instead of introducing this issue into more formulae. We could probably add a separate `ronn` formula to do this.

Reply:
> @carlocab How about solving it by introducing new `ronn` formula and declaring it as a build dependency for listed formulas? 💡

Your message appeared only after I submitted my previous comment. This would be a good solution here, I think.

Reply:
> Your message appeared only after I submitted my previous comment. This would be a good solution here, I think.

No worries 😄  Let's do it...